Core/nbproject/project.properties

```diff
@@ -18,7 +18,7 @@ file.reference.sqlite-jdbc-3.8.11.jar=release/modules/ext/sqlite-jdbc-3.8.11.jar
 file.reference.StixLib.jar=release/modules/ext/StixLib.jar
 file.reference.tika-core-1.14.jar=release/modules/ext/tika-core-1.14.jar
 file.reference.tika-parsers-1.14.jar=release/modules/ext/tika-parsers-1.14.jar
-file.reference.Tsk_DataModel_PostgreSQL.jar=release/modules/ext/Tsk_DataModel_PostgreSQL.jar
+file.reference.sleuthkit-postgresql-4.6.0.jar=release/modules/ext/sleuthkit-postgresql-4.6.0.jar
```

Comment:
Can we use $[VERSION] here as well? Otherwise these files will need to be hand-edited for each TSK release, which would be a bit error prone.